lib/moon_web/pages/contribute_page.ex

```diff
@@ -20,19 +20,89 @@ defmodule MoonWeb.Pages.ContributePage do
       <h1 class="text-moon-32 font-semibold">How to contribute</h1>
-      <PageSection title="First Section">
-        <p class="text-moon-16">Lorem ipsum dolor sit amet</p>
-        <CodeSnippet>mix something</CodeSnippet>
-        <p class="text-moon-16">Lorem ipsum dolor sit amet</p>
-        <CodeSnippet>mix something.else</CodeSnippet>
+      <PageSection title="Setup">
+        <p class="text-moon-16">Clone the repository:</p>
+        <CodeSnippet>git clone git@github.com:coingaming/moon.git</CodeSnippet>
+        <p class="text-moon-16">Start local development server by running:</p>
+        <CodeSnippet>run-locally-dev.sh</CodeSnippet>
+        <p class="text-moon-16">
+          If ./run-locally-dev.sh is not working, try the following steps:</p>
+        <p class="text-moon-16">Install <a
+            href="https://asdf-vm.com/"
+            class="text-piccolo font-medium transition-colors duration-200 hover:text-hit visited:text-hit"
+            target="_blank"
+            rel="noreferrer"
+          >asdf</a> and plugins:</p>
+        <CodeSnippet>asdf plugin add erlang</CodeSnippet>
+        <CodeSnippet>asdf plugin add elixir</CodeSnippet>
+        <CodeSnippet>asdf plugin add nodejs</CodeSnippet>
+        <p class="text-moon-16">Install all the tool versions specified in the .tool_versions file:</p>
+        <CodeSnippet>asdf install</CodeSnippet>
+        <p class="text-moon-16">Install assets dependencies:</p>
+        <CodeSnippet>cd assets && npm i && cd -</CodeSnippet>
+        <p class="text-moon-16">Try running local development server:</p>
+        <CodeSnippet>iex -S mix phx.server</CodeSnippet>
+        <p class="text-moon-16">Stop the command above, then try running:</p>
+        <CodeSnippet>run-locally-dev.sh</CodeSnippet>
+        <p class="text-moon-16">If you do see error in ElixirLS extension "no elixir command found", you'll need to:</p>
+        <CodeSnippet>cp .tool-versions ~</CodeSnippet>
+        <p class="text-moon-16">Install dependencies:</p>
+        <CodeSnippet>mix deps.get</CodeSnippet>
       </PageSection>
-      <PageSection title="Second Section">
+      <PageSection title="Assets repository setup">
         <p class="text-moon-16">
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam sed quam at augue varius accumsan id in sapien. Suspendisse maximus tortor at urna blandit, et varius nibh maximus. Suspendisse potenti. Interdum et malesuada fames ac ante ipsum primis in faucibus.
-        </p>
-        <CodeSnippet>iex -S mix phx.something</CodeSnippet>
-        <p class="text-moon-16">Lorem ipsum dolor sit amet</p>
-        <CodeSnippet>iex -S mix phx.another</CodeSnippet>
+          The following script imports Figma assets and icons. You need to set the values in the .env file. Please make sure you have read access to the assets tool repo https://github.com/coingaming/assets .</p>
+        <CodeSnippet>run-auto-importers.sh</CodeSnippet>
```

Comment:
@tkullisaar @phgrey Could/should we also have per code block CodeSnippet?

Reply:
my mistake, will fix in next PR